tools/importTool/src/main/scala/BytesAccumulatorActor.scala

```diff
@@ -0,0 +1,29 @@
+import java.nio.file.{Files, Paths}
+
+import akka.actor._
+
+
+
+case class Message(value: Int)
+case object LastOffset
+case class FileSize(value:Long)
+
+class BytesAccumulatorActor() extends Actor {
+  var count = 0
```

Comment:
`count` should be `Long`. `Int` is limited only to 2GB

Reply:
done